diff -r 4f745c394d31 include/libtcod.h
--- a/include/libtcod.h	Fri Mar 22 12:39:57 2013 +1300
+++ b/include/libtcod.h	Tue Mar 25 16:22:26 2014 +1300
@@ -110,12 +110,14 @@
 #endif
 
 /* base types */
+#ifndef TCOD_SKIP_ELEMENTARYTYPES
 typedef unsigned char uint8;
 typedef char int8;
 typedef unsigned short uint16;
 typedef short int16;
 typedef unsigned int uint32;
 typedef int int32;
+#endif
 /* int with the same size as a pointer (32 or 64 depending on OS) */
 typedef long intptr;
 typedef unsigned long uintptr;
